internal/evaluator/__testdir__/b.rego

```diff
@@ -0,0 +1,28 @@
+# B set of policies
+package b
+
+# METADATA
+# custom:
+#   short_name: failure
+deny[result] {
+	result := {
+		"code": "main.failure",
```

Comment:
I think this is highlighting a feature gap. Notice how if this rule fails, the `code` is `main.failure`, but if it succeeds, it would be `b.failure`. It would be confusing for a user to experience this behavior.

It's less of a problem when using policies from ec-policies because we ensure those things match there, but a potential pitfall for anyone writing their own policies.

Reply:
Right, copy-pasta error, I used to have just the `main` package, then I created the `a` and `b` packages. Good catch!